### DISCUSSION TITLE
Are you able to reproduce the end-to-end latency in the official repo?

### DISCUSSION THREAD

**Discussion by mtli** [rwightman/efficientdet-pytorch#192] [Q&A]
Hi, thank you for writing this nice PyTorch implementation of EfficientDet! The [official repo](https://github.com/google/automl/tree/master/efficientdet) says D0's end-to-end latency (CNN + pre- and post-processing) is 10ms without TensorRT. I wonder if your repo can produce similar results?

**Comment by rwightman** [comment]
@mtli batch size 1 latency will likely never be comparable without some significant work or new optimization features in PyTorch. I think around batch 16-32 and higher they are comparable.

**Reply by mtli** [reply]
I am not very familiar with TensorFlow, are you suggesting that PyTorch is slower than TensorFlow in this aspect?
